src/goto-checker/bmc_util.h

```diff
@@ -186,6 +186,8 @@ void run_property_decider(
   "(show-symex-strategies)" \
   "(depth):" \
   "(unwind):" \
+  "(unwind-loops):" \
+  "(unwind-recursion):" \
```

Comment:
These should be documented in the user facing documentation (i.e. help message and cprover-manual).